azure-pipelines.yml

```diff
@@ -73,6 +73,9 @@ jobs:
   pool:
     vmImage: 'windows-latest'
   steps:
+  - script: |
+      git config --global core.autocrlf false
+    displayName: Configuring Git
```

Comment:
nit: maybe something like "updating Git LF settings" so we remember what it does when looking at ADO